src/nouislider.ts

```diff
@@ -168,6 +169,19 @@ export class NouisliderComponent implements ControlValueAccessor, OnInit {
     });
   }
 
+  ngOnChanges(changes: any) {
+    if (this.slider && (changes.min || changes.max)) {
+      setTimeout(() => {
+        this.slider.updateOptions({
+          range: {
+            min: this.min,
+            max: this.max
```

Comment:
Are you willing to accept the patch in the current form? I really want to use the slider with the new capabilities :-).